packages/design-system-react-native/package.json

```
@@ -85,7 +85,7 @@
     "typescript": "~5.2.2"
   },
   "peerDependencies": {
-    "@metamask/design-system-twrnc-preset": "^0.1.0",
+    "@metamask/design-system-twrnc-preset": "^0.0.0",
```

This should match `@metamask/design-system-twrnc-preset` bumped version
```suggestion
    "@metamask/design-system-twrnc-preset": "^0.2.0",
```